sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/data_collector.py

```diff
@@ -8,14 +8,17 @@
 from azure.ai.ml.constants._common import BASE_PATH_CONTEXT_KEY
 from azure.ai.ml.entities._deployment.destination import Destination
 from azure.ai.ml.entities._deployment.request_logging import RequestLogging
-from azure.ai.ml.entities._deployment.sampling_strategy import SamplingStrategy
+from azure.ai.ml.entities._deployment.request import Request
+from azure.ai.ml.entities._deployment.response import Response
 
 
 class DataCollector:
     """Data Capture deployment entity
 
-    :param enabled: Is data capture enabled.
-    :type enabled: bool
+    :param request: Will request logging be accepted.
```

Comment:
The description of this param makes me think it's expecting a boolean. 
Are we trying to say that, if this request object is passed, then request logging will be enabled per the definition of this object? 
Same for response

Reply:
Then that means that i can pass an object but I mark `enabled=False`? 
To clarify, my comment is that the param description is not really correct. It isn't just passing the object means logging is accepted, it is that it is accepted and also defines the behavior of the logging (else this should be a pure boolean)